src/com/lightdev/app/shtm/DialogShell.java

```diff
@@ -67,7 +67,7 @@ class DialogShell extends JDialog implements ActionListener {
     /** result value for a confirmed operation */
     public static final int RESULT_OK = 0;
     /** id of associated help topic (if any) */
-    protected String helpTopicId = null;
+    protected String helpTopicId;
```

Comment:
Here and in all similar cases: I prefer to keep explicit default values in field / variable initialization because otherwise, it is not obvious if you forgot it or not and which value is assigned.